sbi/utils/get_nn_models.py

```diff
@@ -101,9 +105,13 @@ def likelihood_nn(
         model: The type of density estimator that will be created. One of [`mdn`,
             `made`, `maf`, `nsf`].
         z_score_theta: Whether to z-score parameters $\theta$ before passing them into
-            the network.
+            the network, can take one of the following:
+            - `none`, None: do not z-score
+            - `independent`: z-score each dimension independently
+            - `structured`: treat dimensions as related, therefore compute mean and std
+            over the entire batch, instead of per-dimension.
```

Comment:
The docstring comments i made above especially apply to these user-facing methods